datasets/_wikidata/categories/wd_categories.yml

```diff
@@ -43,12 +43,18 @@ lookups:
           - member states of the United Nations
           - member of the United Nations Security Council
         value: zz
-      - match: Pakistan Occupied Kashmir
+      - match: 
+          - Pakistan Occupied Kashmir
+          - Kashmir
```

Comment:
let's leave this as a warning for the moment until a new ftm is out with a placeholder country code for this disputed territory, rather than defaulting to one of the claimants